src/Amplitude.js

```diff
@@ -145,6 +151,15 @@ var constructor = function () {
 
     function onUserIdentified(user) {
         var userIdMissing;
+
+        function setUserId(userId) {
+            if (userId) {
+                return getInstance().setUserId(userId);
```

Comment:
@rmi22186  What is the expected return of this method?

Reply:
i'll simplify this.  seems like it's not necessary to return given `onUserIdentified` is a `void` function and doesn't return anything